### ISSUE TITLE
Relative indent on click

### ISSUE THREAD

**Issue by bsdev90** [jemc/vscode-implicit-indent#1]
Hi,
Coming from PhpStorm, it's a great plugin that adds a very practical function.
Is it possible to do the same behavior at the click of the mouse? I sometimes go to a line with the mouse because it's faster than with the keyboard.

**Comment by jemc** [comment]
I agree and would also like to have this.

However, so far I am not sure if and how it is possible to do this with the VSCode API.

If anyone has ideas about how this could be done, please reach out here in the ticket.

**Comment by DevEstacion** [comment]
Found a way to do it, the project you have here wasn't building so I ended up forking and then deleted the tests aswell as a few dependencies.

Check the changes that i made.
https://github.com/DevEstacion/vscode-implicit-indent/blob/master/src/extension.ts

Also, I published the extension on the market and called it indent-on-empty-line, i just want to use it ASAP but i can remove it when you make the update.

@jemc @BenDavidson90
